### PR TITLE
Easter egg blog

### PR DESCRIPTION
**Pull request by alexfrancoeur**
## Summary

-

### Blog Release Checklist

- \[ ] Release on: \<mm/dd/yy>
- \[ ] Published on dev.to (connected to Xata org)
- \[ ] OG image included
- \[ ] Social media is scheduled for this post

### CODE REVIEW

**Comment by miukimiu** [comment]
@alexfrancoeur I think the image you're using as a cover is fine. But here's an alternative, in case you want to show all five easter eggs.

xata-launch-week-promo@2x

**Comment by alexfrancoeur** [comment]
Thanks @miukimiu! Added it in

File: easter-egg-hunt.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: 'Xata Launch Week Easter Egg Hunt'
```
Comment (snide) [inline easter-egg-hunt.mdx:2]:
```suggestion
title: 'Xata launch week Easter egg hunt'
```

**Review by snide — APPROVED** [review]
Sentence case titles

**Pull request merged**